rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -2137,7 +2137,9 @@ namespace rsx
 				//Find zeta address in bound zculls
 				for (const auto& zcull : zculls)
 				{
-					if (zcull.bound)
+					if (zcull.bound &&
```

Comment:
Did you verify that zcull stats don't work when format or AA does not match? I did not see this test case.

Reply:
it's a documented feature.